packages/design-tokens/tokens/color/background.json

```diff
@@ -14,42 +12,32 @@
         "neutral": { "value": "{color.core.neutral.2.value}" },
         "success": { "value": "{color.core.green.1.value}" },
         "warning": { "value": "{color.core.orange.1.value}" },
-        "info": { "value": "{color.core.blue.1.value}" },
         "danger": { "value": "{color.core.red.1.value}" }
       },
 
       "avatar": {
-        "default": { "value": "{color.core.neutral.2.value}" },
+        "default, disabled": { "value": "{color.core.neutral.2.value}" },
```

Comment:
this file needs to contain valid JSON, so comma separated keys aren't going to work. This would instead need to be, 
```
 "default": { "value": "{color.core.neutral.2.value}" },
 "disabled": { "value": "{color.core.neutral.2.value}" },
```

Reply:
I knew it seemed too good to be true... updated.